config/application.rb

```diff
@@ -33,6 +33,13 @@ class Application < Rails::Application
     config.time_zone = "Central Time (US & Canada)"
     config.eager_load_paths << Rails.root.join("lib")
 
+    # Active Storage
+    if ENV["CLOUDFLARE_STORAGE_FEATURE"] == "true"
```

Comment:
can't use `Feature.` here because application hasn't finished starting yet

Reply:
Ohhh, shoot. Hmm.  I think then still add it to Feature and options.yml, just in case other parts of the app need to check it, and then here just repeat the “ENV || application…” which you put in the options.yml